taskcluster/docker/skopeo/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM debian:12-slim
+MAINTAINER Release Engineering <release@mozilla.com>
```

Comment:
nit: MAINTAINER is deprecated, this should be:
```
LABEL maintainer="Release ..."
```

Reply:
Will fix, thanks.  (I copied this from taskgraph itself, but then made a couple of changes, so might as well get this in too.)